apps/site/src/views/Home/sections/Landing/Landing.tsx

```diff
@@ -7,6 +7,7 @@ function Landing() {
 	return (
 		<div className={styles.landing}>
 			<h1>ZotHacks 2023</h1>
+			<p>November 4-5, 2023</p>
```

Comment:
Nitpick suggestion: although we don't have a strict style guide (separate discussion), date ranges have usually been expressed on our sites with an en-dash (`&ndash;`, –) instead of a hyphen.